packages/configure-splash-screen/src/cli-command.ts

```diff
@@ -0,0 +1,196 @@
+import { Command } from 'commander';
+import path from 'path';
+
+import configureAndroid from './android';
+import configureIos from './ios';
+import {
+  SplashScreenImageResizeMode,
+  SplashScreenStatusBarStyle,
+  Platform,
+  PlatformType,
+} from './constants';
+import { AndroidSplashScreenJsonConfig, IosSplashScreenJsonConfig } from './types';
+import { validateEnumValue } from './validators';
+
+const AVAILABLE_OPTIONS_NAMES = [
+  'imageResizeMode',
+  'platform',
+  'backgroundColor',
+  'imagePath',
+  'darkModeBackgroundColor',
+  'darkModeImagePath',
+  'statusBarHidden',
+  'statusBarStyle',
+  'darkModeStatusBarStyle',
+  'statusBarTranslucent',
+  'statusBarBackgroundColor',
+  'darkModeStatusBarBackgroundColor',
+] as const;
+
+type CLIOptionName = typeof AVAILABLE_OPTIONS_NAMES[number];
+
+type CLIOptions = Partial<Record<CLIOptionName, string>>;
+
+interface Configuration {
+  platform: PlatformType;
+  android: AndroidSplashScreenJsonConfig;
+  ios: IosSplashScreenJsonConfig;
+}
+
+async function action(configuration: Configuration) {
+  const { platform, android, ios } = configuration;
+  const rootDir = path.resolve();
```

Comment:
I think `process.cwd()` would be more obvious that `path.resolve()` without argument since that seems to be what we want here
```suggestion
  const rootDir = process.cwd();
```